site/content/extend/plugins/developer-workflow.md

```diff
@@ -16,6 +16,7 @@ subsection: Plugins
 - `make disable` - Disables the plugin on the Mattermost server
 - `make reset` - Disables and re-enables the plugin on the Mattermost server
 - `make debug-plugin` - Starts a `delve` process and attaches it to your running plugin
+- `make clean` - Force deletes the content of build related files. Use when running into build issues,
```

Comment:
Typo
```suggestion
- `make clean` - Force deletes the content of build related files. Use when running into build issues.
```

Reply:
Agree. The folder names could change. It is okay to keep it generic.

Reply:
`build related` should also be `build-related`

Reply:
Heads up that there's an unresolved request above to change the comma to a period.